tests/test_config.py

```diff
@@ -257,20 +286,25 @@ def test_log_config_file(mocked_logging_config_module):
 
 
 @pytest.fixture(params=[0, 1])
-def web_concurrency(request):
+def web_concurrency(request: Any) -> Iterator[int]:
     yield request.param
     if os.getenv("WEB_CONCURRENCY"):
         del os.environ["WEB_CONCURRENCY"]
 
 
 @pytest.fixture(params=["127.0.0.1", "127.0.0.2"])
-def forwarded_allow_ips(request):
+def forwarded_allow_ips(request: Any) -> Iterator[str]:
```

Comment:
not sure about this, worth checking if Any could be avoided
```suggestion
def forwarded_allow_ips(request: FixtureRequest) -> Iterator[str]:
```

Reply:
I agree, it's good to be more specific than `Any`. `pytest.FixtureRequest` works. The `param` attribute is [optional](https://docs.pytest.org/en/latest/reference/reference.html#pytest.FixtureRequest), so mypy requires a check for the attribute before indexing into it, like `getattr(request, "param")`.